docs/Troubleshooting.md

```diff
@@ -192,6 +192,23 @@ Jest takes advantage of new features added to Node 6. We recommend that you upgr
 
 Make sure you are not using the `babel-plugin-istanbul` plugin. Jest wraps Istanbul, and therefore also tells Istanbul what files to instrument with coverage collection. When using `babel-plugin-istanbul`, every file that is processed by Babel will have coverage collection code, hence it is not being ignored by `coveragePathIgnorePatterns`.
 
+## Defining Tests
+
+Tests much be defined synchronously for Jest to be able to collect your tests.
+
+This means when you are using `test.each` you cannot set the table asynchronously within a `beforeEach` / `beforeAll`.
```

Comment:
IMO we should have the example above the mention of `.each`.

Reply:
done in 716bc5f :)